server/app/attestation-service-ctl/readme.md

```diff
@@ -0,0 +1,129 @@
+- get-policy
+- set-policy
+- get-reference-data
+- set-reference-data
+- restore-default-policy
+- restore-default-reference-data
+- test-attestation
+
+### get-policy
+
+Get the TEE-specific evaluation `Policy(.rego)` from Attestation Server.
+```SHELL
+./attestation-service-ctl get-policy --tee <TEE> [--addr <ADDR>] [--output <PATH>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): Designate the attestation server's `management-sock` address. Default is `127.0.0.1:3001`.
+- output(optional): Output file path to write the policy. Default is `./Policy.rego`.
+
+### set-policy
+
+Set the TEE-specific evaluation `Policy(.rego)` according to the content of `POLICY`.
+```SHELL
+./attestation-service-ctl set-policy --tee <TEE> --policy <POLICY> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- policy: The path of local `Policy(.rego)` which will be upload to Attestation Server and saved in the AS user's workdir: `/opt/attestation-service/<TEE>/`.
+- addr(optional): Designate the Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### get-reference-data
+
+Get the TEE-specific `Reference Data(.json)` from Attestation Server.
+```SHELL
+./attestation-service-ctl get-reference-data --tee <TEE> [--addr <ADDR>] [--output <PATH>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): The Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+- output(optional): Output file path to write the reference data. Default is `./Reference_data.json`.
+
+### set-reference-data
+
+Set the TEE-specific evaluation `Reference Data(.json)` according to the content of `REFERENCE_DATA`.
+```SHELL
+./attestation-service-ctl set-reference-data --tee <TEE> --reference-data <REFERENCE_DATA> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- reference-data: The path of local `Reference Data(.json)` which will be upload to Attestation Server and saved in the AS user's workdir: `/opt/attestation-service/<TEE>/`.
+- addr(optional): Designate the Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### restore-default-policy
+
+Restore the Attestation Server's TEE-specific `Policy(.rego)` to default.
+```SHELL
+./attestation-service-ctl restore-default-policy --tee <TEE> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): The Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### restore-default-reference-data
+
+Restore the Attestation Server's TEE-specific `Reference Data(.json)` to default.
+```SHELL
+./attestation-service-ctl restore-default-reference-data --tee <TEE> [--addr <ADDR>]
+```
+- tee: Specify the target TEE name.
+- addr(optional): The Attestation Server's `management-sock` address. Default is `127.0.0.1:3001`.
+
+### test-attestation
+
+Test Attestation Server's `attestation` function with the input `EVIDENCE` and print the attestation result to stdout.
```

Comment:
Why not simply call that command `attest`? I can see it potentially becoming more than just sending a sample evidence.